packages/wonder-blocks-data/src/util/gql-types.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+/**
+ * A GraphQL operation.
+ */
+export type GqlOperation<
```

Comment:
Will webapp eventually use this version of this type?

Reply:
For the things that use this implementation of things, yes.

Reply:
Sounds good. 🙂